### ISSUE TITLE
[Feature] Circulation program

### ISSUE THREAD

**Issue by Speed1** [erikkastelec/hass-WEM-Portal#67]
Hi @erikkastelec 

Thanks for the great plugin.
Do you see also the possibility to integrate the circulation program of the hot water? So that it could be disabled when nobody is at home?

Thx

**Comment by erikkastelec** [comment]
Is this setting available inside the mobile app?

**Comment by Speed1** [comment]
Yes, in the program tab you can configure the circulation program by day and time. It would be great if there is a way to turn it on or off with a single button.

![IMG_2427](https://user-images.githubusercontent.com/11215021/235966750-ea8db9ce-bd73-4c40-b886-f52bd01e96d7.jpeg)

**Comment by erikkastelec** [comment]
I will make an effort to implement this, but I cannot provide a specific timeline at this moment.

**Comment by dilki001** [comment]
This will be a great feature . 
thanks

**Comment by Speed1** [comment]
Hi @erikkastelec 
Do you have any news on this? Still work in progress or not possible?

Thanks 😃

**Comment by ghenzler** [comment]
I think making the circulation program be controllable is a bit of an effort (it'll need the same UI as the APP in home assistant, not sure if there are some ready-made UI components for that). 

I created #94 for a simpler alternative to achieve what you need @Speed1 (while I think having this feature would also be nice, it'll just more work to get it done)

**Comment by erikkastelec** [comment]
Circulation program implementation indeed is one of the things that turned out to be harder to implement than I initially thought. I will leave it on the roadmap, but I will probably prioritize other features.
